packages/dashboard/src/app/views/myStories/index.js

```diff
@@ -129,14 +133,45 @@ function MyStories() {
     [getAuthors, setQueriedAuthors]
   );
 
+  let queryCategoriesBySearch = useCallback(
+    (categorySearchTerm) => {
+      return getCategories(categorySearchTerm).then((data) => {
+        if (!isMounted.current) {
+          return;
+        }
+
+        const categoryData = data.map(({ id, name }) => ({
+          id,
+          name,
+        }));
+        setQueriedCategories((current) => {
+          const existingIds = current.map(({ id }) => id);
+          const newCategories = categoryData.filter(
+            (category) => !existingIds.includes(category.id)
```

Comment:
Linter called this out, but probably just need to rename category to something else here potentially